js/util-galery.js

```diff
@@ -0,0 +1,68 @@
+
+const bigPhoto = document.querySelector('.big-picture');
+const body = document.querySelector('.body');
+const commentCounter = document.querySelector('.social__comment-count');
+const commentsLoader = document.querySelector('.comments-loader');
+const closeButton = document.querySelector('.big-picture__cancel  cancel');
+const socialComments = document.querySelector('.social__comments');
+
+const hideBigPhoto = () => {
+  bigPhoto.classList.add('hidden');
+  body.classList.remove('modal-open');
+  commentCounter.classList.remove('hidden');
+  commentsLoader.classList.remove('hidden');
+  //document.removeEventListener('keydown', onDocumentKeydown);
+  //closeButton.removeEventListener('click', onCloseButton);
+};
+
+const onDocumentKeydown = (evt) => {
+  if (evt.key === 'Escape') {
+    evt.preventDefault();
+    hideBigPhoto();
+  }
+};
+
+const onCloseButton = () => {
+  hideBigPhoto();
+};
+
+const renderPhotoDetails = ({url, description, likes, comments}) => {
+  bigPhoto.querySelector('.big-picture__img').src = url;
+  bigPhoto.querySelector('.social__caption').textContent = description;
+  bigPhoto.querySelector('.likes-count').textContent = likes;
+  document.querySelector('.comments-count').textContent = comments.length;
+};
+
+//сюда вставить рендер комментов
+const createComment = ({avatar, name, message}) => {
+  const comment = document.createElement('li');
+  comment.innerHTML = '<img class="social__picture" src="" alt="" width="35" height="35"> <p class="social__text"></p>';
+  comment.querySelector('.social__picture').src = avatar;
+  comment.querySelector('.social__picture').alt = name;
+  comment.querySelector('.social__text').textContent = message;
+};
```

Comment:
В конце функции нужно вернуть наш заполненный комментарий.
```suggestion
  return comment;
};
```
Т.е. функция создает один коммент, заполняет его данными и отдает этот заполненный элемент, чтобы мы могли его где-то добавить